src/web/View.tsx

```diff
@@ -24,6 +25,31 @@ const _styles = {
         flex: '0 0 auto',
         overflow: 'hidden',
         alignItems: 'stretch'
+    },
+
+    resizeDetectorContainerStyles: {
```

Comment:
This is really clever. But like most clever code, it really demands some good comments to explain to the reader what it does and why.